### PR TITLE
Add and update kickstart templates

### PR DESCRIPTION
**Pull request by ajurjevi**

### CODE REVIEW

File: templates/CentOS-7.6-x86_64.ks.erb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+$SNIPPET('pre_install_network_config')
+# enable installation monitoring
+$SNIPPET('pre_anamon')
+
+#
+# packages list
+#
+%packages
+
+@base
```
Comment (jsosic) [inline templates/CentOS-7.6-x86_64.ks.erb]: Can you remove `@base`, that would eliminate the need to disable all those services?

This is kinda minimal install for CentOS 7 (`@core` replaces `@base`):

```
@core
net-tools
xfsprogs
-firewalld
-firewalld-filesystem
-alsa-firmware
-alsa-tools-firmware
-iwl*firmware
-ivtv-firmware
-aic49xx-firmware
-crystalhd-firmware
-plymouth*
-wpa_supplicant
-NetworkManager
-NetworkManager-team
-NetworkManager-tui
-NetworkManager-wifi
```

File: templates/settings.erb [hunk 2]
```diff
@@ -220,7 +220,7 @@ puppetca_path: "<%= @puppetca_path %>"
 # installed is removed from the puppet master server if the puppet
 # master server is running on the same machine as cobbler. This
 # requires puppet_auto_setup above to be enabled
-remove_old_puppet_certs_automatically: <%= @remove_old_puppet_certs %>
+remove_old_puppet_certs_automatically: <%= @sign_puppet_certs_automatically %>
```
Comment (jsosic) [inline templates/settings.erb]: This won't work cause the name of the param in `init.pp` isn't updated to match this value.

**Review by jsosic — CHANGES_REQUESTED** [review]
Can you please split this PR into couple of smaller ones, so that each one has an atomic commit and corresponding commit message?

Suggestion of prs:
* PR1: add new kickstarts and update existing ones
* PR2: update dhcp template
* PR3: ...

Each commit/pr should be a single unit that addresses one and one thing only, and not a bundled commit that addresses 10 different issues in one commit.

File: templates/CentOS-7.6-x86_64.ks.erb (hunk 1 above)
Comment (eimamagi) [reply templates/CentOS-7.6-x86_64.ks.erb]: fine by me

**Review by jsosic — APPROVED** [review]

**Pull request merged**